src/services/userService.js

```diff
@@ -0,0 +1,47 @@
+const { createUniqueID } = require('../utils/createUniqueID');
+
+let users = [];
+
+const initUsers = () => {
+  users = [];
+};
+
+const getAllUsers = () => {
+  return users;
+};
+
+const getUserById = (id) => {
```

Comment:
You use like userService.[your method] so we already know that it related to users

```suggestion
const getById = (id) => {
```